components/sinks/cu_zenoh_ros_sink/README.md

```diff
@@ -0,0 +1,66 @@
+# cu-zenoh-ros-sink
+
+This is a Copper sink task that publishes messages to ROS 2 via Zenoh middleware.
+It allows seamless integration between Copper applications and ROS 2 nodes by using the [RMW zenoh implementation](https://github.com/ros2/rmw_zenoh).
+
+## Configuration
+
+The sink requires the following configuration parameters:
+
+- zenoh_config_file: Zenoh [configuration json file](https://github.com/ros2/rmw_zenoh/blob/rolling/rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_SESSION_CONFIG.json5) (optional).
+- domain_id: ROS domain ID (optional, defaults to "0"),
+- namespace: The ROS namespace (optional, defaults to "copper").
+- node: The ROS node name (optional, defaults to "node").
+- topic: The ROS topic name to publish to (optional, defaults to "copper").
+
+Example configuration in a Copper application:
+
+```ron
+(
+    tasks: [
+        (
+            id: "publisher",
+            config: {
+                "zenoh_config_file": "/opt/ros/humble/share/rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_SESSION_CONFIG.json5",
```

Comment:
That breaks the "self-contained" nature of the Copper config.

We do an include! of the RON file at compile time to be able to have a 1 file deployment and a path to a no_std deployment too: ie. *no* filesystem at all.

is it possible, maybe in another PR to provide an embedded version of it: for example a config_str: and the inline version of the json file?